### ISSUE TITLE
[LICENSE] Please add a license

### ISSUE THREAD

**Issue by SoftwareRat** [powenn/AltServer-LinuxGUI#19]
Hello,
I want to play with this project but sadly it doesn't have a license?

Can you please add a LICENSE file which explains how we can use your project?

Thanks :)

**Comment by powenn** [comment]
Indeed,I should add license according to AltServer Linux's License and AltServer's , following AGPL3.0 , so basically you should make your project being open source and remember to credit related projects and their owner, And enjoy do something to your fork or anything else , could I ask you what you want to do?

**Comment by SoftwareRat** [comment]
> Indeed,I should add license according to AltServer Linux's License and AltServer's , following AGPL3.0 , so basically you should make your project being open source and remember to credit related projects and their owner, And enjoy do something to your fork or anything else , could I ask you what you want to do?

I want to make fedora package on copr

**Comment by tommipontinen76** [comment]
> > Indeed,I should add license according to AltServer Linux's License and AltServer's , following AGPL3.0 , so basically you should make your project being open source and remember to credit related projects and their owner, And enjoy do something to your fork or anything else , could I ask you what you want to do?
> 
> I want to make fedora package on copr

That would be great!!
